### ISSUE TITLE
U2F authn fails after SP1 and SP2 installation

### ISSUE THREAD

**Issue by nynymike** [GluuFederation/oxAuth#362]
After using SP1, U2F authentication fails.

```
2016-10-28 20:52:27,695 ERROR [org.xdi.oxauth.service.external.ExternalAuthenticationService]
Traceback (most recent call last):
  File "<iostream>", line 146, in prepareForStep
AttributeError: 'org.jboss.seam.security.Credentials' object has no attribute 'getUser'

        at org.python.core.Py.AttributeError(Py.java:205)
....
at org.xdi.oxauth.service.external.ExternalAuthenticationService.executeExternalPrepareForStep(ExternalAuthenticationService.java:213)
```

From script log

```
2016-10-29 04:06:09,765 INFO  [org.xdi.service.PythonService] (ajp-bio-127.0.0.1-8009-exec-12) U2F. Prepare for step 2. Failed to determine user name
```

Must be an error in this part of the script:

```
            authenticationService = AuthenticationService.instance()
            user = authenticationService.getAuthenticatedUser()
            if (user == None):
                print "U2F. Prepare for step 2. Failed to determine user name"
                return False
```

**Comment by yurem** [comment]
merged in 2.4.4.2: https://github.com/GluuFederation/oxAuth/commit/532933d3f38d90c0082eeaa50bf2ad6f597aabd7

**Comment by yurem** [comment]
Fixed
